### ISSUE TITLE
[HL25] New hud scaled too big at 768p

### ISSUE THREAD

**Issue by iodream** [ValveSoftware/halflife#3499]
I've noticed a recent update says "The HUD will no longer scale to 2x size at 1024x768 and 1280x720 resolution."

Could this be also done for 1366x768? Or is there some reason why this resolution was left out? It's pretty common for laptops and i'd say it's still way too big

 
![20231120010321_1](https://github.com/ValveSoftware/halflife/assets/42720874/9bedac71-94af-41fb-8cc4-0ab5fcdf872e)

**Comment by Swordman14** [comment]
Yeah, the hud is still scaled 2x in 768, which makes it really big

**Comment by s4molet1k** [comment]
bro this update was so unnecessary, this update was released just to increase hype of half life series, this update broke main menu customization, hud size, and also some of the gsource mods.

i'm also trying to find a solution to this, but still nothing, guess there's no way to fix it, except changing each .spr hud file manually for it to look normal, or going back to pre 25 update release of half life.

**Comment by ph1sh55** [comment]
> bro this update was so unnecessary, this update was released just to increase hype of half life series, this update broke main menu customization, hud size, and also some of the gsource mods.
> 
> i'm also trying to find a solution to this, but still nothing, guess there's no way to fix it, except changing each .spr hud file manually for it to look normal, or going back to pre 25 update release of half life.

You can fix it for yourself without touching a .spr file- in the valve/sprites/hud.txt you can comment out (using // at the beginning of the line) all of  the '1280' resolution lines which tells halflife which sprites to use if the horizontal resolution exceeds 1280.
`//selection		1280 1280/hud_selection	0	0	340	90

//bucket1			1280 1280/hud_bucket1	0	0	40	40
//bucket2			1280 1280/hud_bucket2	0	0	40	40		
//bucket3			1280 1280/hud_bucket3	0	0	40	40		
//bucket4			1280 1280/hud_bucket4	0	0	40	40		
//bucket5			1280 1280/hud_bucket5	0	0	40	40		
//bucket0			1280 1280/hud_bucket0	0	0	40	40		
//etc...
`
Once all of the 1280 lines are commented out, then it should defer to the 640 resolution rows in the same file which have a smaller hud.
